.snyk

```diff
@@ -5,7 +5,7 @@ ignore:
   SNYK-JAVA-CAJULIUSDAVIES-30073:
     - '*':
         reason: Fix not available
-        expires: 2019-11-30T19:21:05.816Z
+        expires: 2019-12-15T00:00:00.000Z
```

Comment:
Might be worth giving this a longer lifetime? A couple of months perhaps?

Reply:
I'd rather have us address it more often. If its a deal-breaker I can bump it to a month or two

Reply:
Fair do's. The original ignore was added 5 months ago, so I reckon we'll end up bumping quite a lot though.

Reply:
If fix is not available then I think it's fine to wait a few months. We don't want the builds to constantly break because of this